import-automation/executor/app/executor/import_executor.py

```diff
@@ -351,6 +351,78 @@ def _get_latest_version(self, import_dir: str) -> str:
+                template_mcf[0]))[0]
+            input_files.extend(template_mcf)
+
+        cleaned_csv = []
+        if pattern := import_input.get('cleaned_csv'):
+            cleaned_csv = glob.glob(os.path.join(absolute_import_dir, pattern))
+            input_files.extend(cleaned_csv)
+
+        node_mcf = []
+        if pattern := import_input.get('node_mcf'):
+            node_mcf = glob.glob(os.path.join(absolute_import_dir, pattern))
+            input_files.extend(node_mcf)
+            if not import_prefix:
+                import_prefix = os.path.splitext(os.path.basename(
+                    node_mcf[0]))[0]
+        return input_files, import_prefix
+
+    def _invoke_import_tool(self, absolute_import_dir: str,
+                            relative_import_dir: str, version: str,
+                            import_spec: dict):
+        """ 
+        Invokes DC import tool to generate resolved mcf.
+        """
+        import_inputs = import_spec.get('import_inputs', [])
+        for import_input in import_inputs:
+
+            input_files, import_prefix = self._get_import_input_files(
+                import_input, absolute_import_dir)
+            if not import_prefix:
+                logging.error(
+                    'Skipping genmcf due to missing import input spec.')
+                continue
+            output_path = os.path.join(absolute_import_dir, import_prefix,
+                                       'validation')
+
+            # Run dc import tool to generate resolved mcf.
+            logging.info(f'Generating resolved mcf for {import_prefix}')
+            import_tool_args = [f'-o={output_path}', 'genmcf']
+            import_tool_args.extend(input_files)
+            process = _run_user_script(
+                interpreter_path='java',
+                script_path='-jar ' + self.config.import_tool_path,
+                timeout=self.config.user_script_timeout,
+                args=import_tool_args,
+                cwd=absolute_import_dir,
+                env=os.environ.copy(),
+            )
+            _log_process(process=process)
+            process.check_returncode()
+            logging.info(
+                f'Generated resolved mcf for {import_prefix} in {output_path}.')
+
+            if not self.config.skip_gcs_upload:
+                # Upload output to GCS.
+                gcs_output = f'{relative_import_dir}/{version}/{import_prefix}/validation'
+                logging.info(
+                    f'Uploading genmcf output to GCS path: {gcs_output}')
+                for filename in os.listdir(output_path):
+                    filepath = os.path.join(output_path, filename)
+                    if os.path.isfile(filepath):
+                        dest = f'{gcs_output}/{filename}'
+                        self.uploader.upload_file(
+                            src=filepath,
+                            dest=dest,
+                        )
```

Comment:
Can we add a `log_metric()` from https://github.com/datacommonsorg/data/blob/master/util/log_util.py#L33 to indicate validation stage status?